src/EngagementPlugin/Editor/EngagementPostBuild.cs

```diff
@@ -608,6 +609,10 @@ public static void OnPostprocessBuild(BuildTarget target, string pathToBuiltProj
 			string CTFramework = "CoreTelephony.framework";
 			Debug.Log("Adding "+ CTFramework + " to XCode Project");
 			pbx.AddFrameworkToProject(targetUID,CTFramework,true);
+
+			string UNFramework = "UserNotifications.framework";
+			Debug.Log("Adding "+ UNFramework + " to XCode Project");
+			pbx.AddFrameworkToProject(targetUID,UNFramework,true);
```

Comment:
Should be "false" here as the UserNotifications framework is optional, not used on devices running iOS < 10